.gitignore

```diff
@@ -1,4 +1,4 @@
-doc/*.html
+/doc/*.html
```

Comment:
Can we pull these changes out into another PR?

Reply:
...do you want them in the same PR as each other?

Reply:
Since I don't need to change this file to match my work environment, I'm rescinding all of these changes.